src/main/java/org/schedulesdirect/api/Program.java

```diff
@@ -1388,4 +1445,40 @@ private String toString(Collection<?> collection, int maxLen) {
 		builder.append("]");
 		return builder.toString();
 	}
+	
+	/**
+	 * @return the entityType
+	 */
+	public EntityType getEntityType() {
+		return entityType;
+	}
+	
+	/**
+	 * @param entityType the entityType to set
+	 */
+	public void setEntityType(EntityType entityType) {
+		this.entityType = entityType;
+	}
+	
+	/**
+	 * @return the keywords
+	 */
+	public Map<String, List<String>> getKeywords() {
```

Comment:
I'm not familiar with the keywords data from upstream.  But I'm just wondering out loud if there shouldn't be an actual KeyWords type to encapsulate this data instead of just using a simple Map?  Not saying it should be done that way, but just thinking out loud.  I'd say it might be a good idea if there are (or could be) other extra things you can do with key words. If not, then a Map is probably good enough.